README.md

```diff
@@ -25,6 +25,10 @@ responsible for the public ONS website.
 
 You can use [websysd](https://github.com/ONSdigital/dp/blob/master/websysd) to run the dp services from a centralised location:
 
+If you are running a publishing stack the global configs require having "ENABLE_PRIVATE_ENDPOINTS=true" and "ENCRYPTION_DISABLED=true” if encryption is not setup.
```

Comment:
Just me being pedantic, but might read a bit better like:
If you are running the publishing stack, then you must set "ENABLE_PRIVATE_ENDPOINTS=true" and "ENCRYPTION_DISABLED=true” if encryption is not setup.